code/baseline.py

```diff
@@ -18,6 +27,7 @@ def encode(self, text):
 
     :param text: string
         The text to be encoded.
+
     :return: list
```

Comment:
This is just some basic tokenization. Feel free to add more logic to make the tokenization more performant.